src/nullObjects.js

```diff
@@ -24,9 +24,13 @@ export const NULL_ACCOUNT: Account = {
 };
 
 export const NULL_USER: User = {
-  fullName: '',
   avatarUrl: '',
   email: '',
+  fullName: '',
```

Comment:
This is a useful change. You can introduce it in a separate commit.

Reply:
Considering you did the change in the other PR, do remove this change even if it is not functionally different.